delta/utils/decode/tf_ctc.py

```diff
@@ -51,6 +51,7 @@ def ctc_decode_last_to_blankid(decode_result, blank_id):
   '''
     change the value of blank_label elements from num_classes - 1 to blank_id
```

Comment:
change the value of blank_label elements from num_classes - 1 to blank_id after removing blank_id by decoder.